sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaSquidSensor.java

```diff
@@ -51,6 +53,8 @@
 @DependedUpon(value = JavaUtils.BARRIER_AFTER_SQUID)
 public class JavaSquidSensor implements Sensor {
 
+  private static final Logger LOG = LoggerFactory.getLogger(JavaSquidSensor.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOG" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AS1068)